src/include/daos_errno.h

```diff
@@ -239,7 +239,10 @@ extern "C" {
 	       TX is not committed)					\
 	/** Agent is incompatible with libdaos */			\
 	ACTION(DER_AGENT_INCOMPAT,	(DER_ERR_DAOS_BASE + 29),	\
-	       Agent is incompatible with libdaos)
+	       Agent is incompatible with libdaos)			\
+	/** Event queue not initialized */			\
+	ACTION(DER_EQ_UNINIT,		(DER_ERR_DAOS_BASE + 30),	\
```

Comment:
Is there any reason why you couldn't use DER_UNINIT here?  This functionality doesn't seem to be over the threshold for requiring a new error number to be added.

Reply:
No, there is no reason at all, I just thought including a new error number would make sense for this particular scenario. @ashleypittman, do you want me to use DER_UNINIT instead?

Reply:
I can't see any reason not to.  Having an extra error number which is similar to an existing one, is only returned in one specific place, and never tested for explicitly doesn't add anything of value to the product.

Reply:
Working on that.